src/private/privateAPIs.ts

```diff
@@ -35,7 +35,10 @@ export function getUserJoinedTeams(
   ensureInitialized();
 
   if (
-    GlobalVars.hostClientType === HostClientType.android &&
+    (GlobalVars.hostClientType === HostClientType.android ||
+      GlobalVars.hostClientType === HostClientType.teamsRoomsAndroid ||
```

Comment:
Why add these here? This change was made to address a very specific issue in Android Mobile Client. I don't think we should be adding more entries here.

Reply:
Originally all the Android platforms, including mobile and devices were sharing the same HostClientType - `android`, so I assume here the we should apply the logic to all the Android platforms to keep the same behavior as before. Do you think it's unnecessary? How about other changes in `authentication.ts` and `media.ts` ?

Reply:
Hmm... Are you saying that all of these are just variations of our Android client? Do you know who owns the Client SDK implementation for these different client types? @shbhadra, can you please chime in here.

Reply:
Hey Edward, is the corresponding change for this checked into the android client? 
The value for `GlobalVars.hostClientType` is set during the initialize() call, and is a value that is sent by the client. 
Android code seems to be still setting this as 'android' always.

Reply:
I have another PR to make the hostClientType change on Android side, it's not merged to develop now. All the new types here are subtype of "android". 
https://domoreexp.visualstudio.com/Teamspace/_git/SkypeSpaces-Android/pullrequest/352658